korge/src/commonTest/kotlin/com/soywiz/korge/view/ViewsTest.kt

```diff
@@ -247,4 +251,98 @@ class ViewsTest : ViewsForTesting() {
         assertEquals(rect.x, 0.0)
         assertEquals(rect.y, -5.0)
     }
+
+    @Test
+    fun testSpriteAnimationPlay1Times() = viewsTest {
```

Comment:
Maybe you could extract animation tests in a new test class (`AnimationTest.kt`, for example)? The `ViewsTest` class is already looks overloaded, and it seems like quite a lot new tests only for animation.

Reply:
Thx. I can do that.